packages/aws-cdk/lib/api/util/sdk.ts

```diff
@@ -113,6 +146,7 @@ export class SDK implements ISDK {
     const environment = await this.resolveEnvironment(account, region);
     return new AWS.CloudFormation({
       ...this.retryOptions,
+      endpoint: this.endpoints.cloudformation,
```

Comment:
You might get a `Cannot get property "cloudformation" of null` error with `--endpoints null`

Reply:
Should be fixed now.